db/migrations/0018_add_lti_usage.py

```diff
@@ -0,0 +1,43 @@
+'''
+Migration for MiVideo media started hourly table
+'''
+
+from yoyo import step
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_placement_types (
+            id INTEGER NOT NULL AUTO_INCREMENT,
+            description VARCHAR(100) NOT NULL,
+            table_name VARCHAR(64) NOT NULL,
+            PRIMARY KEY (id)
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4;
+    '''),
+    step('''
+        INSERT IGNORE INTO lti_placement_types
+        (id, description, table_name) VALUES
+        (0, 'Zoom', 'lti_zoom_placement')
```

Comment:
Yeah, I switched this because I wanted the explicit 0 for this row to be able to reference this in the code later. I could remove the auto increment or just trust that it the order will be predictable.

I only have the IGNORE it in there because we have a pattern of `CREATE TABLE IF NOT EXISTS`. We wouldn't want to insert twice.

The migrations would prevent this from running so we *really* don't need the INSERT IGNORE or IF NOT EXISTS.

Reply:
I'm just going to delete this table and do the enum